### PR TITLE
#52919327 Update muscle list

### PR DESCRIPTION
**Pull request by MuthomiMate**
#### What does this PR do?
Updates the muscle list.
#### Description of Task to be completed?
Add the following muscles as well as their exercises:
- lateral deltoid
- Posterious Deltoid 
- Forearms 
#### How should this be manually tested?
After checking out to this branch run
```
invoke load_fixtures --settings-path=settings.py
```
Run the application and after login to the application under Exercises tab, select Muscle Overview. The updated muscle list with the new muscles and exercises will be shown on the left side of the image.
#### What are the relevant pivotal tracker stories?
[#152919327](https://www.pivotaltracker.com/story/show/152919327)
#### Screenshots (if appropriate)
image
image

@ckwagaba @VincentHokie @jonathankamau

### CODE REVIEW

File: .bowerrc [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-    "directory" : "wger/static/bower_components",
+    "directory" : "wger/core/static/bower_components",
     "interactive": false
-}
+}
```
Comment (jonathankamau) [inline .bowerrc]: A new line is needed here.

**Comment by MuthomiMate** [comment]
@jonathankamau  done

File: wger/exercises/fixtures/muscles.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [
     {
-        "pk": 2, 
+        "pk": 2,
```
Comment (jonathankamau) [inline wger/exercises/fixtures/muscles.json]: There is a tab space here, after 2.

**Comment by jk** [comment]
@MuthomiMate You mentioned the wrong handle

**Comment by VincentHokie** [comment]
😂 😂 😂 @MuthomiMate

**Review by Sharonsyra — COMMENTED** [review]
Nice job.

**Pull request merged**